UnityProject/Assets/Scripts/Util/RandomUtils.cs

```diff
@@ -4,6 +4,14 @@
 using UnityEngine;
 using Random = UnityEngine.Random;
 
+public enum SpinMode
```

Comment:
feels weird to have this here- It is RandomUtils as in utilities for handling randomization

Reply:
where would be the best place to put it?

Reply:
somewhere closer to where it is used, which I assume is the UniversalPhysics thingy. Maybe create a new file for all enums it uses? Like we do for health.

Reply:
could do